tests/src/main/resources/META-INF/mods.toml

```diff
@@ -29,7 +29,8 @@ license="LGPL v2.1"
 
 [[mods]]
     modId="finalize_spawn_fix_test"
-
+[[mods]]
+    modId="villager_change_profession_event_test"
```

Comment:
remove these lines. You did not add a test mod under this modid afaik

Reply:
Also, please test your PR since this line would've crashed. If you do not know how to test, please ask in the discord for help. Neo testing is quite tricky

Reply:
Ty for reminding me, I originally did create a class, but moved the test method to `EntityEventTests`.